package io.github.mklkj.kommunicator.data.exceptions

import io.ktor.utils.io.errors.IOException

class AlreadyExist : IOException()
